drivers/ethernet/eth_native_posix.c

```diff
@@ -444,7 +444,7 @@ static void eth_iface_init(struct net_if *iface)
 
 	ctx->dev_fd = eth_iface_create(ctx->if_name, false);
 	if (ctx->dev_fd < 0) {
-		LOG_ERR("Cannot create %s (%d)", ctx->if_name, ctx->dev_fd);
+		LOG_ERR("Cannot create %s (%d)", ctx->if_name, -errno);
```

Comment:
Maybe don't negate it either, as POSIX errno's are positive.

Reply:
As this is not a posix call, I prefer that the value is <0 to indicate an error, just like we do in other parts of the system.

Reply:
I wondered that eth_iface_create() actually returns errno of the underlying POSIX system, and my concern was exactly to not mix it up with Zephyr-level error codes. But ok.